src/test/java/uk/gov/digital/ho/pttg/api/HmrcResourceIntegrationTest.java

```diff
@@ -372,6 +377,35 @@ public void shouldGiveUnauthorizedWhenFailedUnauthorizedRetriesWithHmrc() throws
         assertThat(responseEntity.getStatusCode()).isEqualTo(UNAUTHORIZED);
     }
 
+    @Test
+    public void shouldRetryAccessCallIfConnectionRefused() {
+        // given
+        mockService.reset();
+
+        mockService
+                .expect(requestTo(containsString("/audit")))
+                .andExpect(method(POST))
+                .andRespond(withSuccess());
+
+        mockService
+                .expect(times(3), requestTo(containsString("/access")))
```

Comment:
Where does the magic number 3 come from?  Should this be set up as part of the test?